src/core/pubnub-common.ts

```diff
@@ -2692,10 +2704,8 @@ export class PubNubCore<
       let { channels, channelGroups } = parameters;
 
       // Remove `-pnpres` channels / groups if they not acceptable in the current PubNub client configuration.
-      if (!this._configuration.getKeepPresenceChannelsInPresenceRequests()) {
-        if (channelGroups) channelGroups = channelGroups.filter((channelGroup) => !channelGroup.endsWith('-pnpres'));
-        if (channels) channels = channels.filter((channel) => !channel.endsWith('-pnpres'));
-      }
+      if (channelGroups) channelGroups = channelGroups.filter((channelGroup) => !channelGroup.endsWith('-pnpres'));
```

Comment:
will this be applicable to ee and non ee case both cases? OR it's just for getSubscribedChannels/Groups method?